perun-openapi/openapi.yml

```diff
@@ -5863,6 +6235,225 @@ paths:
         default:
           $ref: '#/components/responses/ExceptionResponse'
 
+  /json/registrarManager/getFormItems/vo:
+    get:
+      tags:
+        - RegistrarManager
+      operationId: getFormItemsForVo
+      summary: Gets all items in VO application form.
+      parameters:
+        - $ref: '#/components/parameters/voId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfApplicationFormItemsResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/registrarManager/getFormItems/vo-type:
+    get:
+      tags:
+        - RegistrarManager
+      operationId: getFormItemsForVoWithType
+      summary: Gets items of specified type in VO application form, for initital registration or extension of account.
+      parameters:
+        - $ref: '#/components/parameters/voId'
+        - { name: type, schema: { type: string }, in: query, description: "application type: INITIAL or EXTENSION"}
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfApplicationFormItemsResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/registrarManager/getFormItems/group:
+    get:
+      tags:
+        - RegistrarManager
+      operationId: getFormItemsForGroup
+      summary: Gets all items in Group application form.
+      parameters:
+        - $ref: '#/components/parameters/groupId'
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfApplicationFormItemsResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/registrarManager/getFormItems/group-type:
+    get:
+      tags:
+        - RegistrarManager
+      operationId: getFormItemsForGroupWithType
+      summary: Gets items of specified type in Group application form, for initital registration or extension of account.
+      parameters:
+        - $ref: '#/components/parameters/groupId'
+        - { name: type, schema: { type: string }, in: query, description: "application type: INITIAL or EXTENSION"}
+      responses:
+        '200':
+          $ref: '#/components/responses/ListOfApplicationFormItemsResponse'
+        default:
+          $ref: '#/components/responses/ExceptionResponse'
+
+  /json/registrarManager/updateFormItems/vo:
```

Comment:
When the `json` serializer/deserializer is used and the `POST` method is called, you should pass the data in the `requestBody` instead of parameters. There is one exception.

This can be done by specifying the `urlinjsonout` serializer/deserializer, but it should be used only when you are passing ids (e.g. calling deletion of some objects). 

In this case, it would be better to define the `requestBody` instead of the parameters. 

Please check all of your methods if they satisfy these conditions. If you have any questions, ask me or Makub.

Reply:
Thank you, I did not know that. I hope it is better now.